tools/DevicesApiTester/Commands/Spi/WriteRandomBytes.cs

```diff
@@ -30,22 +30,20 @@ public int Execute()
                 Mode = Mode,
             };
 
-            using (var spi = CreateSpiDevice(connectionSettings))
+            using (var spiDevice = CreateSpiDevice(connectionSettings))
             {
                 // Write random bytes of data
                 var buffer = new byte[ByteCount];
                 new Random().NextBytes(buffer);
 
                 Console.WriteLine($"Writing random bytes:{Environment.NewLine}{HexStringUtilities.FormatByteData(buffer)}");
-                spi.Write(buffer.AsSpan());
+                spiDevice.Write(buffer.AsSpan());
             }
 
             return 0;
         }
-
-        [Option('n', "byte-count", HelpText = "The number of random bytes to write to the connection", Required = false, Default = 16)]
-        public int ByteCount { get; set; }
 
-        private static readonly Random s_random = new Random();

```

Comment:
instead of removing this it makes sense to just use this instance everywhere

Reply:
> instead of removing this it makes sense to just use this instance everywhere

I don't quite follow this?  The 2 `s_random` variables I removed were currently not used.

`new Random()` is still used once in I2cWriteRandomBytes.cs and once in [Spi]WriteRandomBytes.cs.

Reply:
what I'm saying is basically: `s/new Random/s_random/` and add it `s_random` back

Reply:
I apologize, but I'm still a little confused on this feedback.

So from what I can see `s_random` was never referenced.  A warning from VS was how I noticed this when I removed.

Since there is only two locations/commands that use code related to generating random bytes, I don't really see the need to have a static Random object compared to what is used below...

```csharp
// Write random bytes of data
var buffer = new byte[ByteCount];
new Random().NextBytes(buffer);
```

I also don't understand the syntax for `s/new Random/s_random/`.

Again, sorry for being a little slow to pick up on this.  Please provide a little more code detail and where to store `s_random` if it going to be referenced in multiple command types (like `Random().NextBytes` is currently with I2C and SPI).

Reply:
`s/new Random/s_random/` means replace `new Random` with `s_random`

`new Random().NextBytes(buffer);` => `s_random.NextBytes(buffer)`

fine to leave as is though if it's only used in two places